src/System.Runtime.InteropServices.RuntimeInformation/src/OSPlatform.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public struct OSPlatform : IEquatable<OSPlatform>
+    {
+        private readonly string _osPlatformString;
+
+        private const string WindowsString = "WINDOWS";
+        private const string LinuxString = "LINUX";
+        private const string OSXString = "OSX";
```

Comment:
See also https://github.com/dotnet/corefx/issues/1576 where this was discussed.

"FreeBSD" is the desired identifier.